src/Service/Resolvers/MsSqlQueryBuilder.cs

```diff
@@ -120,16 +120,14 @@ public string Build(SqlUpsertQueryStructure structure)
             }
             else
             {
-                return $"SET TRANSACTION ISOLATION LEVEL SERIALIZABLE;BEGIN TRANSACTION; UPDATE {QuoteIdentifier(structure.DatabaseObject.SchemaName)}.{QuoteIdentifier(structure.DatabaseObject.Name)} " +
-                    $"WITH(UPDLOCK) SET {Build(structure.UpdateOperations, ", ")} " +
+                return $"UPDATE {QuoteIdentifier(structure.DatabaseObject.SchemaName)}.{QuoteIdentifier(structure.DatabaseObject.Name)} " +
```

Comment:
When comparing to serializable isolation level, this update would have blocked on a concurrent long running read. With this PR, it will not. Is that desired? Can we document this change in behavior please?

Reply:
With Read Committed isolation level, the update transaction would not get blocked. This is because, the shared read lock gets released after the row is read. In Serializable isolation level, the lock is held for the entire transaction causing the update to block. 

Since, the update transaction does not get blocked, more concurrent executions should be possible. In this scenario, since there no multiple SELECT statements, things like non-repeatable reads and phantom reads are not applicable.  

So, switching from Serializable to Read Committed should not cause any harm